examples/async_io/async.cpp

```diff
@@ -0,0 +1,7 @@
+#include <hpx/lcos/future.hpp>
+using namespace hpx;
+int main() {
+  future<int> f = async([]() -> int { return 91; });
+  int val = f.get();
+}
+
```

Comment:
1. `using namespace hpx` is not good style (to avoid namespace clashes).
2. `main` needs to `return hpx::finalize();`.
3. you need to include `hpx/async.hpp`
4. you need to include `hpx/hpx_main.hpp`
5. I really, *really*, encourage you to build and run these examples on your own. You'll learn more about HPX in the process and you'll be able to write better documentation after it.